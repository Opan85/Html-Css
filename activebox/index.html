<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:700|Raleway:300,400,600,700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/animate.css">
    <script src="https://kit.fontawesome.com/34fc356e08.js"></script>
    <title>ActiveBox</title>
</head>
<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <div class="header__logo">
                    <a href="#" class="logo__link link"><img src="img/header/ActiveBox.png" alt="Logo" class="img__logo"></a> 
                </div>
                <nav class="nav">
                    <ul class="nav__bar">
                        <li class="nav__item"><a href="#features" class="nav__link link">features</a></li>
                        <li class="nav__item"><a href="#works" class="nav__link link">works</a></li>
                        <li class="nav__item"><a href="#team" class="nav__link link">our team</a></li>
                        <li class="nav__item"><a href="#testotimonials" class="nav__link link">testotimonials</a></li>
                        <li class="nav__item"><a href="#download" class="nav__link link">download</a></li>
                    </ul>
                </nav>
            </div> <!--Header inner-->
            <div class="header__text-box">
                <h1 class="header__title">Your favorite one page  milti purpose template</h1>
                <p class="header__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent commodo cursus magna vel scelerisque nisl consectetur et.</p>
                <a href="#" class="header__btn button link">find our more</a>
            </div>
        </div> <!--Container-->
    </header>
    <section id="features" data-wow-duration="2s" class="features wow fadeInLeft">
        <div class="container">
            <div class="features__inner">
                <div class="features__item">
                    <img src="img/features/1.png" alt="" class="features__img">
                    <h3 class="features__title">Easily customised</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
                <div class="features__item">
                    <img src="img/features/2.png" alt="" class="features__img">
                    <h3 class="features__title">Responsive ready</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
                <div class="features__item">
                    <img src="img/features/3.png" alt="" class="features__img">
                    <h3 class="features__title">Modern design</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
                <div class="features__item">
                    <img src="img/features/4.png" alt="" class="features__img">
                    <h3 class="features__title">Clean code</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
                <div class="features__item">
                    <img src="img/features/5.png" alt="" class="features__img">
                    <h3 class="features__title">Ready to ship</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
                <div class="features__item">
                    <img src="img/features/6.png" alt="" class="features__img">
                    <h3 class="features__title">Download to free</h3>
                    <p class="features__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus auctor fringilla.</p>
                </div>
            </div> <!--Fuatures__inner-->
        </div> <!--Container-->
    </section>

    <section id="works" data-wow-duration="2s" class="works wow slideInRight">
        <div class="works__inner">
            <div class="works__img--1 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--2 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--3 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--4 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--5 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--6 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--7 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
            <div class="works__img--8 bg">
                <div class="works__info">
                    <h3 class="works__title">Project name</h3>
                    <p class="works__text">User interface design</p>
                </div>
            </div>
        </div> <!-- works__inner-->
    </section>

    <section id="team" data-wow-duration="2s" class="team wow fadeIn">
        <div class="container">
            <div class="team__inner">
                <div class="team__card">
                    <div class="img-box">
                        <img src="img/team/1.jpg" alt="" class="card__img">
                    </div>
                    <div class="card__info">
                        <h3 class="card__title">Ruth Woods</h3>
                        <p class="card__subtitle">Founder, CEO</p>
                        <p class="card__text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social__icon">
                        <div class="social__inner">
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-facebook-f"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-twitter"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-vk"></i></a></div>
                            <div class="icon"><a class="icon__link " href="#"><i class="fab fa-pinterest-square"></i></a></div>
                        </div>
                    </div>
                </div> <!-- team__card-->
                <div class="team__card">
                    <div class="img-box">
                        <img src="img/team/2.jpg" alt="" class="card__img">
                    </div>
                    <div class="card__info">
                        <h3 class="card__title">Timothy Reed</h3>
                        <p class="card__subtitle">Co-founder, Developer</p>
                        <p class="card__text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social__icon">
                        <div class="social__inner">
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-facebook-f"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-twitter"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-vk"></i></a></div>
                            <div class="icon"><a class="icon__link " href="#"><i class="fab fa-pinterest-square"></i></a></div>
                        </div>
                    </div>
                </div> <!-- team__card-->
                <div class="team__card">
                    <div class="img-box">
                        <img src="img/team/3.jpg" alt="" class="card__img">
                    </div>
                    <div class="card__info">
                        <h3 class="card__title">Victoria Valdez</h3>
                        <p class="card__subtitle">UI Designer</p>
                        <p class="card__text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social__icon">
                        <div class="social__inner">
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-facebook-f"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-twitter"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-vk"></i></a></div>
                            <div class="icon"><a class="icon__link " href="#"><i class="fab fa-pinterest-square"></i></a></div>
                        </div>
                    </div>
                </div> <!-- team__card-->
                <div class="team__card">
                    <div class="img-box">
                         <img src="img/team/4.jpg" alt="" class="card__img">
                    </div>
                    <div class="card__info">
                        <h3 class="card__title">Beverly Little</h3>
                        <p class="card__subtitle">Data Scientist</p>
                        <p class="card__text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social__icon">
                        <div class="social__inner">
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-facebook-f"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-twitter"></i></a></div>
                            <div class="icon"><a class="icon__link" href="#"><i class="fab fa-vk"></i></a></div>
                            <div class="icon"><a class="icon__link " href="#"><i class="fab fa-pinterest-square"></i></a></div>
                        </div>
                    </div>
                </div> <!-- team__card-->
            </div> <!--team__inner-->
        </div> <!--container-->
    </section>

    <section id="testotimonials" class="testotimonials">
        <div class="testotimonials__inner">
            <div class="image">
                <img src="img/testotimonials/1.jpg" alt="" class="img-slider">
            </div>
            <div class="image__content">
                <div class="slider__content">
                    <p class="slider__text">"Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam..."</p>
                    <p class="content__autor">SUSAN SIMS, INTERACTION DESIGNER AT XYZ</p>
                </div>
            </div>
        </div>
    </section>

    <section id="download" class="download">
        <div class="container">
            <div class="download__inner">
                <p class="download__text">Are You Ready to Start? Download Now For Free!</p>
                <p class="download__subtext">FUSCE DAPIBUS, TELLUS AC CURSUS COMMODO</p>
                <a href="#" data-wow-duration="1.5s" data-wow-delay="1s" class="download__btn button link wow bounce">download here</a>
            </div>
        </div>
    </section>

    <footer class="footer">
        <div class="container">
            <div class="footer__inner">
                <div class="footer__location">
                    <h3 class="location__title footer__title">Location</h3>
                    <p class="location__text footer__text">3481 Melrose Place<br>Beverly Hills, CA 90210</p>
                </div>
                <div class="footer__share">
                    <h3 class="share__title footer__title">Share whith love</h3>
                    <div class="social__inner social__footer">
                        <div class="icon footer__icon"><a class="icon__link footer-icon__link" href="#"><i class="fab fa-facebook-f"></i></a></div>
                        <div class="icon footer__icon"><a class="icon__link footer-icon__link" href="#"><i class="fab fa-twitter"></i></a></div>
                        <div class="icon footer__icon"><a class="icon__link footer-icon__link" href="#"><i class="fab fa-vk"></i></a></div>
                        <div class="icon footer__icon"><a class="icon__link footer-icon__link" href="#"><i class="fab fa-pinterest-square"></i></a></div>
                    </div>
                </div>
                <div class="footer__about">
                    <div class="footer__about--inner">
                        <h3 class="about__title footer__title">About Activebox</h3>
                        <p class="about__text footer__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec ullamcorper nulla non metus auctor fringilla.</p>
                    </div>
                </div>
            </div> <!--Footer__inner-->
        </div> <!--Container-->
        <div class="subfooter">
            <div class="subfooter__inner">
                <p class="copyright">Copyright &#169 2015 ActiveBox. All Rights Reserved;</p>
                <p class="copyright">Made with <i class="fas fa-heart"></i><a class="subfooter__link" href="#">by Kamal Chaneman</a> </p>
            </div>
        </div>
    </footer>


<script src="js/wow.min.js"></script>
<script>new WOW().init();</script>  
</body>
</html>