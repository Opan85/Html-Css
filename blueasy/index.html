<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <script src="https://kit.fontawesome.com/34fc356e08.js"></script>
    <title>Blueasy</title>
</head>

<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <nav class="nav">
                    <a href="#" class="logo__link link">
                        <h2 class="logo">Blu<span>e</span>asy</h2>
                    </a>
                    <ul class="menu">
                        <li class="menu__list"><a href="#" class="menu__link active link">home</a></li>
                        <li class="menu__list"><a href="#services" class="menu__link link">services</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link">features</a></li>
                        <li class="menu__list"><a href="#portfolio" class="menu__link link">portfolio</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link">contact</a></li>
                    </ul>
                </nav>
                <div class="header__content">
                    <h2 class="header__title">“I’m looking for the unexpected.<br> I’m looking for things I’ve never
                        seen before."</h2>
                </div>
            </div>
            <!--Header__inner-->
        </div>
        <!--Container-->
    </header>
    <main>

        <section id="services" class="services">
            <div class="container">
                <div class="section__title--box">
                    <h2 class="section__title">Services</h2>
                </div>
                <div class="services__inner">
                    <div class="services__item">
                        <div class="item__bg bg--1"></div>
                        <div class="services__info">
                            <h3 class="info__title">Coffee</h3>
                            <div class="info__text">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet
                                rollliquoris bear claw.Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu
                                applicake jujubes.</div>
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="item__bg bg--2"></div>
                        <div class="services__info">
                            <h3 class="info__title">Instant</h3>
                            <div class="info__text">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet
                                rollliquoris bear claw.Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu
                                applicake jujubes.</div>
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="item__bg bg--3"></div>
                        <div class="services__info">
                            <h3 class="info__title">Serious</h3>
                            <div class="info__text">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet
                                rollliquoris bear claw.Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu
                                applicake jujubes.</div>
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="item__bg bg--4"></div>
                        <div class="services__info">
                            <h3 class="info__title">Frame</h3>
                            <div class="info__text">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet
                                rollliquoris bear claw.Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu
                                applicake jujubes.</div>
                        </div>
                    </div>
                </div>
                <!--Services__inner-->
            </div>
            <!--Container-->
        </section>

        <section id="portfolio" class="portfolio">
            <div class="container">
                <nav class="section__nav nav">
                    <div class="section__title black">
                        <h2 class="section__title black">Portfolio</h2>
                    </div>
                    <ul class="section__menu menu">
                        <li class="menu__list"><a href="#"
                                class="menu__link link--color active active--color link">all</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link--color link">panoramas</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link--color link">portraits</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link--color link">macro</a></li>
                        <li class="menu__list"><a href="#" class="menu__link link--color link">journal</a></li>
                    </ul>
                </nav>
                <div class="portfolio__inner">
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/1.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/2.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/3.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/4.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/5.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/6.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/7.webp" alt="pictures" class="potfolio__img">
                    </div>
                    <div class="portfolio__item">
                        <div class="plus"><i class="fas fa-plus"></i></div>
                        <img src="img/portfolio/8.webp" alt="pictures" class="potfolio__img">
                    </div>
                </div>
                <!--portfolio__inner-->
            </div>
            <!--Container---->
        </section>

        <section class="default">
            <div class="container">
                <div class="default__inner">
                    <div class="default__item">
                        <div class="section__title--box">
                            <h2 class="section__title">Just default Section</h2>
                        </div>
                        <p class="item__text">Bear claw marzipan bear claw applicake I love muffin. Lemon drops gummi
                            bears pastry gummi bears sesam snaps. I love underwear.com. Souffle cotton candy dessert
                            candy ice cream wafer gummies cheesecake brownie.<br><br>Muffin chupa chups jelly beans
                            sweet
                            pie applicake Croissant chocolate cake I like pudding. Ice cream a love powder pudding apple
                            pie marshmallow. Cupcake marzipan oat cake bonbon I love candy canes toffee</p>
                        <a href="#" class="item__link link">Visit me</a>
                    </div>
                    <div class="default__item--bg"></div>

                </div>
            </div>
        </section>

        <div class="twitter">
            <div class="container">
                <div class="twitter__inner">
                    <div class="section__title--box">
                        <h2 class="section__title">Twitter</h2>
                    </div>
                    <div class="twitter__icon"><i class="fab fa-twitter"></i></div>
                    <p class="twitter__text">Unerdwear cookie liquorice. Cake donut cupcake lollipop soufflé candy.
                        Chocolate oat cake <a href="#" class="twitter__link link">@cheesecake</a></p>
                </div>
            </div>
        </div>

        <section class="whois">
            <div class="container">
                <div class="section__title--box">
                    <h2 class="section__title">Who is john doe?</h2>
                </div>
                <div class="whois__inner">
                    <div class="whois__item">
                        <p class="item__text">Bear claw marzipan bear claw applicake I love muffin. Lemon drops gummi
                            bears pastry gummi bears sesam snaps. I love underwear.com. Souffle cotton candy dessert
                            candy ice cream wafer gummies cheesecake brownie.<br><br>Muffin chupa chups jelly beans
                            sweet
                            pie applicake Croissant chocolate cake I like pudding. Ice cream a love powder pudding apple
                            pie marshmallow. Cupcake marzipan oat cake bonbon I love candy canes toffee</p>
                    </div>
                    <div class="whois__item">
                        <p class="item__text">Pudding dessert jujubes tiramisu gingerbread croissant tiramisu applicake. Sesame snaps sugar plum cotton candy chocolate bonbon lemon drops candy canes cotton candy. Cake toffee pie bear claw pastry</p>
                        <div class="icon__wrapper">
                            <div class="icon">
                                <a href="#" class="icon__link link--1 link"><i class="fab fa-twitter"></i></a> </div>
                            <div class="icon">
                                <a href="#" class="icon__link link--2 link"><i class="fab fa-google-plus-g"></i></a> </div>
                            <div class="icon">
                                <a href="#" class="icon__link link--3 link"><i class="fas fa-basketball-ball"></i></a> </div>
                            <div class="icon">
                                <a href="#" class="icon__link link--4 link"><i class="fas fa-coffee"></i></a> </div>
                        </div>
                    </div>

                </div>
            </div>
        </section>


    </main>
</body>

</html>