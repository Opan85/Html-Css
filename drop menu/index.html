<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,900&display=swap" rel="stylesheet">
    <title>Drop down menu</title>
</head>
<body>
    <header>
        <nav class="nav">
            <ul class="menu">
                <li class="menu-list"><a href="#" target="blank" class="menu-link">Home</a></li>
                <li class="menu-list"><a href="#" target="blank" class="menu-link">Photo</a>
                    <ul class="submenu">
                        <li class="submenu-list"><a href="#" class="submenu-link">More1</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More2</a></li>
                    </ul>
                </li>
                    
                <li class="menu-list"><a href="#" target="blank" class="menu-link">Blog</a>
                    <ul class="submenu">
                        <li class="submenu-list"><a href="#" class="submenu-link">More1</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More2</a></li>
                    </ul>
                </li>
                    
                <li class="menu-list"><a href="#" target="blank" class="menu-link">About</a>
                    <ul class="submenu">
                        <li class="submenu-list"><a href="#" class="submenu-link">More1</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More2</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More3</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More4</a></li>
                    </ul>
                </li>
                   
                <li class="menu-list"><a href="#" target="blank" class="menu-link">Contacts</a>
                    <ul class="submenu">
                        <li class="submenu-list"><a href="#" class="submenu-link">More1</a></li>
                        <li class="submenu-list"><a href="#" class="submenu-link">More2</a></li>
                    </ul>
                </li>
                    
            </ul>
        </nav>
    </header>
    
</body>
</html>