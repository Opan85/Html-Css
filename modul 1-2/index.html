<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700,900&display=swap" rel="stylesheet">
    <title>Modul 1</title>
</head>
<body>
    <header class="header">
        <nav class="nav-bar">
            <ul class="menu">
                <li class="list-item"> <a class="link-item" href="#">Lifestyle</a></li>
                <li class="list-item"> <a class="link-item" href="#">Photodiary</a></li>
                <li class="list-item"> <a class="link-item" href="#">Music</a></li>
                <li class="list-item"> <a class="link-item" href="#">Travel</a></li>
            </ul>
        </nav>
    </header>
    <main>
        <section class="content">
            <div class="wrap">
                <article class="content-item">
                    <img src="img/image1.jpg" alt="">
                    <p class="subtitle">Lifestyle</p>
                    <h2 class="title">More then just a music festival</h2>
                    <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. A sit vel reiciendis porro aspernatur amet repellendus, nisi odit impedit repudiandae sunt laudantium dolore, adipisci ipsum laborum nesciunt culpa consectetur recusandae!</p>
                </article>
                <article class="content-item">
                    <img src="img/image2.jpg" alt="">
                    <p class="subtitle">Photodiary</p>
                    <h2 class="title">Life tested better whith coffee</h2>
                    <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. A sit vel reiciendis porro aspernatur amet repellendus, nisi odit impedit repudiandae sunt laudantium dolore, adipisci ipsum laborum nesciunt culpa consectetur recusandae!</p>
                </article>
                <article class="content-item">
                    <img src="img/image3.jpg" alt="">
                    <p class="subtitle">Music</p>
                    <h2 class="title">American dream</h2>
                    <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. A sit vel reiciendis porro aspernatur amet repellendus, nisi odit impedit repudiandae sunt laudantium dolore, adipisci ipsum laborum nesciunt culpa consectetur recusandae!</p>
                </article>
                <article class="content-item">
                    <img src="img/image4.jpg" alt="">
                    <p class="subtitle">Travel</p>
                    <h2 class="title">A day exploring the Alps</h2>
                    <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. A sit vel reiciendis porro aspernatur amet repellendus, nisi odit impedit repudiandae sunt laudantium dolore, adipisci ipsum laborum nesciunt culpa consectetur recusandae!</p>
                </article>
            </div>
            <div class="button">
                <button class="btn">load more</button>
            </div>
        </section>
       
    </main>
    
</body>
</html>

