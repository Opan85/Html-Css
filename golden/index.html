<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <script src="https://kit.fontawesome.com/34fc356e08.js"></script>
    <title>Golden</title>
</head>

<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <div class="logo__box">
                    <a href="#" class="logo__link">
                        <img src="img/logo/logo.png" alt="logo" class="logo__img image">
                    </a>
                </div>
                <nav class="nav">
                    <ul class="menu">
                        <li class="menu__list"><a href="#" class="menu__link link">home</a></li>
                        <li class="menu__list"><a href="#services" class="menu__link link">services</a></li>
                        <li class="menu__list"><a href="#portfolio" class="menu__link link">portfolio</a></li>
                        <li class="menu__list"><a href="#about" class="menu__link link">about</a></li>
                        <li class="menu__list"><a href="#contact" class="menu__link link">contact</a></li>
                    </ul>
                </nav>
            </div>
            <!--Header__inner-->
            <div class="header__info">
                <p class="header__uptitle">Welcome To Our Studio!</p>
                <h2 class="header__title">It’s nice to Meet you</h2>
                <a href="#" class="header__btn link">tell me more</a>
            </div>
        </div>
        <!--Container-->
    </header>

    <section id="services" class="services">
        <div class="container">
            <div class="section__info">
                <h2 class="section__title">Services</h2>
                <p class="section__subtitle">Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="services__inner">
                <div class="services__item">
                    <h3 class="item__title"> E-Commerce</h3>
                    <p class="item__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quasi excepturi
                        placeat neque mollitia sapiente temporibus ipsam est earum rerum doloremque.</p>
                    <div class="item__bg bg--1"></div>
                </div>
                <div class="services__item">
                    <h3 class="item__title">Responsive Web</h3>
                    <p class="item__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quasi excepturi
                        placeat neque mollitia sapiente temporibus ipsam est earum rerum doloremque.</p>
                    <div class="item__bg bg--2"></div>
                </div>
                <div class="services__item">
                    <h3 class="item__title">Web security</h3>
                    <p class="item__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quasi excepturi
                        placeat neque mollitia sapiente temporibus ipsam est earum rerum doloremque</p>
                    <div class="item__bg bg--3"></div>
                </div>
            </div>
            <!--Services__inner-->
        </div>
        <!--Container-->
    </section>

    <section id="portfolio" class="portfolio">
        <div class="container">
            <div class="section__info">
                <h2 class="section__title">Our portfolio</h2>
                <p class="section__subtitle">Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="portfolio__inner">
                <div class="portfolio__item">
                    <img src="img/portfolio/img1.png" alt="pictures" class="portfolio__img image">
                </div>
                <div class="portfolio__item">
                    <img src="img/portfolio/img2.png" alt="pictures" class="portfolio__img image">
                </div>
                <div class="portfolio__item">
                    <img src="img/portfolio/img3.png" alt="pictures" class="portfolio__img image">
                </div>
            </div>
        </div>
    </section>

    <section id="about" class="section about">
        <div class="container">
            <div class="section__info">
                <h2 class="section__title">About us</h2>
                <p class="section__subtitle">Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="about__inner">
                <div class="about__item">
                    <img src="img/about/about1.jpg" alt="pictures" class="about__img">
                </div>
                <div class="about__item">
                    <img src="img/about/about2.jpg" alt="pictures" class="about__img">
                </div>
                <div class="about__item">
                    <img src="img/about/about3.jpg" alt="pictures" class="about__img">
                </div>
                <div class="about__item">
                    <img src="img/about/about4.jpg" alt="pictures" class="about__img">
                </div>
                <div class="about__item">
                    <h4 class="date__about--item">July 2010</h4>
                    <p class="subtitle__about--item">Our Humble Beginnings</p>
                    <p class="text__about--item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex nostrum
                        illo illum mollitia minima, atque fugit ratione reiciendis laborum at corporis neque! Eaque hic
                        ad, in sunt architecto corrupti fuga?</p>

                </div>
                <div class="about__item">
                    <h4 class="date__about--item">January 2011</h4>
                    <p class="subtitle__about--item">Facing Startup Battles</p>
                    <p class="text__about--item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex nostrum
                        illo illum mollitia minima, atque fugit ratione reiciendis laborum at corporis neque! Eaque hic
                        ad, in sunt architecto corrupti fuga?</p>

                </div>
                <div class="about__item">
                    <h4 class="date__about--item">December 2012</h4>
                    <p class="subtitle__about--item">Enter The Dark Days</p>
                    <p class="text__about--item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex nostrum
                        illo illum mollitia minima, atque fugit ratione reiciendis laborum at corporis neque! Eaque hic
                        ad, in sunt architecto corrupti fuga?</p>

                </div>
                <div class="about__item">
                    <h4 class="date__about--item">february 2014</h4>
                    <p class="subtitle__about--item">Our Triumph</p>
                    <p class="text__about--item">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex nostrum
                        illo illum mollitia minima, atque fugit ratione reiciendis laborum at corporis neque! Eaque hic
                        ad, in sunt architecto corrupti fuga?</p>
                </div>

            </div>
            <!--about__inner-->
        </div>
        <!--Container-->
    </section>

    <footer class="footer">
        <div class="container">
            <div class="footer__inner">
                <div class="footer__box">
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-instagram"></i>
                        </a>
                    </div>
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-facebook-f"></i>
                        </a>
                    </div>
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-twitter"></i>
                        </a>
                    </div>
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-pinterest-p"></i>
                        </a>
                    </div>
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-vk"></i>
                        </a>
                    </div>
                    <div class="footer__item">
                        <a href="#" class="footer__link link">
                            <i class="fab fa-google-plus-g"></i>
                        </a>
                    </div>
                </div>

                <div class="footer__box">
                    <div class="footer__info">
                        © Copyright 2014 FreebiesXpress.com
                    </div>
                </div>
            </div>
            <!--footer__inner-->
        </div>
        <!--Container-->
    </footer>

</body>

</html>